app/views/groups/client_group_list.scala.html

```diff
@@ -138,8 +140,7 @@
         @if(form.errors.map(error => error.message).contains("error.select-clients.empty")) {
         <div class="govuk-form-group govuk-form-group--error">
             <p id="clients-error" class="govuk-error-message">
-                <span class="govuk-visually-hidden">Error:</span>
-                @msgs("error.select-clients.empty")
+                <span class="govuk-visually-hidden">Error:</span> @msgs("error.select-clients.empty")
```

Comment:
also I just realised we should be using `@msgs("error-prefix")` here instead of Error: